src/elements/content-sidebar/activity-feed/common/user-link/UserLink.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { Link } from '../../../../../components/link';
+import { GetProfileUrlCallback } from '../../../../common/flowTypes';
+import './UserLink.scss';
+
+export interface UserLinkProps {
+    className?: string;
+    getUserProfileUrl?: GetProfileUrlCallback;
+    id: string;
+    name: string;
+}
+
+interface State {
+    profileUrl?: string;
+}
+
+class UserLink extends React.PureComponent<UserLinkProps, State> {
+    state: State = {
+        profileUrl: undefined,
+    };
```

Comment:
You don't need to initialize undefined properties
```suggestion
    state: State = {};
```